invenio/modules/access/local_config.py

```diff
@@ -24,12 +24,14 @@
 
 # pylint: disable=C0301
 
-from invenio.config import CFG_SITE_NAME, CFG_SITE_URL, CFG_SITE_SECURE_URL, CFG_SITE_SUPPORT_EMAIL, CFG_SITE_RECORD, CFG_SITE_ADMIN_EMAIL
 from invenio.base.i18n import _
-from invenio.base.globals import cfg as config
+from invenio.base.config import CFG_SITE_URL, CFG_SITE_RECORD, \
```

Comment:
I can't because returns:
`RuntimeError: working outside of application context`
Can I use some lazy import?